server/api/rounds.py

```diff
@@ -448,23 +478,34 @@ def draw_sample(election: Election, round: Round, sample_sizes: Dict[str, int]):
     )
 
     # Create RoundContest objects to include the contests in this round
-    for contest in contests_that_havent_met_risk_limit:
-        round_contest = RoundContest(
-            round_id=round.id,
-            contest_id=contest.id,
+    round.round_contests = [
```

Comment:
Double checking that we don't need to `db_session.add(round)` after this?

Reply:
doesn't seem like it? if the RoundContests didn't get created successfully, everything else downstream would break, so i think it's working